spec/models/donation_spec.rb

```diff
@@ -9,16 +9,27 @@
   it { is_expected.to validate_numericality_of(:amount).is_greater_than(0) }
 
   describe ".search" do
-    let(:smithie_black) { create(:donor, name: "Smithie Black") }
-    let(:bob_smith_white) { create(:donor, name: "Bob Smith-White") }
-    let(:john_smith) { create(:donor, name: "John Smith") }
-    let(:carl_grey) { create(:donor, name: "Carl Grey") }
+    before do
```

Comment:
Then we are facing the same problem with lets that they are created lazily (the one we had last night).
How can we do it without using lets like in .search?